src/main/java/com/savvato/tribeapp/services/ToBeReviewedCheckerServiceImpl.java

```diff
@@ -111,11 +111,7 @@ public void updateTables(ToBeReviewed tbr, boolean hasMatchingRejectedPhrase, bo
             toBeReviewedRepository.setHasBeenGroomedTrue(tbr.getId());
         }
         else {
-            if (!hasMatchingRejectedPhrase) {
```

Comment:
Now that line 114 has been removed, `hasMatchingRejectedPhrase` is an unused variable. Clean up.